Docs/README-csharp-v2.md

```diff
@@ -0,0 +1,255 @@
+# Usage examples - C&#35;
+
+## Basics
+
+The PhaxioClient class is the entry point for any Phaxio operation (you'll also need to add ```using Phaxio.V2```).
+
+    var phaxio = new PhaxioV2Client(key, secret);
```

Comment:
The trouble is that we'd then need to namespace that client somehow - there are several methods that are the same function but have different method signatures now (such as listing account numbers). We'd also get a situation where some methods wouldn't work since they're not available in the selected version.

We could change the client to Phaxio (so V1 would be PhaxioClient and V2 would be Phaxio).